beta-3-setup.md

```diff
@@ -1392,12 +1393,71 @@ also work (albeit with a self-signed certificate):
 
     https://hello-openshift.cloudapps.example.com
 
+## Project Administration
+
+If `joe` now wants to let `alice` look at his project, with his project
+administrator rights he can add her using the `osadm policy` command:
+
+    [joe]$ osadm policy add-role-to-user view alice
+
+Now login at the command line as `alice` to see what is available:
+
+    osc login -u alice
+    Authentication required for https://ose3-master.example.com:8443 (openshift)
+    Password:  <redhat>
+    Login successful.
+    
+    Using project "demo"
+
+`alice` has no projects of her own yet, so she is automatically configured
+to look at the `demo` project. She has "view" access, so `osc status`
+and `osc get pods` and so forth should show her the same thing as
+`joe`. However, she cannot make changes:
+
+    [alice]$ osc get pods
+    POD                       IP         CONTAINER(S)      IMAGE(S)
+    hello-openshift-1-zdgmt   10.1.2.4   hello-openshift   openshift/hello-openshift
+    [alice]$ osc delete pod hello-openshift-1-zdgmt
+    Error from server: "/api/v1beta1/pods/hello-openshift-1-zdgmt?namespace=demo" is forbidden because alice cannot delete on pods with name "hello-openshift-1-zdgmt" in demo
+
+Also login as `alice` in the web console and confirm that she can view
+the `demo` project.
+
+`joe` could also give `alice` the role of `edit`, which gives her access
+to do nearly anything in the project except adjust access.
+
+    [joe]$ osadm policy add-role-to-user edit alice
+
+Now she can delete that pod if she wants, but she can not add access for
+another user or upgrade her own access. To allow that, `joe` could give
+`alice` the role of `admin`, which gives her the same access as himself.
+
+    [joe]$ osadm policy add-role-to-user admin alice
+
+There is no "owner" of a project, and projects can certainly be created
+without any administrator. `alice` or `joe` can remove the `admin`
+role (or all roles) from each other or themselves at any time without
+affecting the existing project.
+
+    [joe]$ osadm policy remove-user joe
```

Comment:
I thought it was funny to have joe self-delete. Alice gets to delete the project...